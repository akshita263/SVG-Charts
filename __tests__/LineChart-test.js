import React from 'react';
import 'react-native';
import LineChart from '../Screens/LineChart';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<LineChart />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});