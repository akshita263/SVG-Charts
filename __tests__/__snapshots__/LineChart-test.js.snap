// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "alignSelf": "center",
        "fontSize": 30,
        "fontWeight": "bold",
        "padding": 20,
      }
    }
  >
    LINE CHART
  </Text>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderWidth": 5,
        "margin": 20,
        "opacity": 1,
        "padding": 10,
      }
    }
  >
    <View
      style={
        Object {
          "height": 250,
        }
      }
    >
      <View
        onLayout={[Function]}
        style={
          Object {
            "flex": 1,
          }
        }
      />
    </View>
  </View>
</View>
`;
